components/VC/MosipVCItem/MosipVCItem.tsx

```diff
@@ -73,7 +73,6 @@ export const MosipVCItem: React.FC<
               vcMetadata={props.vcMetadata}
               verifiableCredential={verifiableCredential}
               emptyWalletBindingId={emptyWalletBindingId}
-              onPress={() => props.onPress(service)}
```

Comment:
why this is removed?

Reply:
onPress is not being used anywhere in the activation, Hence removing it.